spring-integration-webflux/src/test/java/org/springframework/integration/webflux/outbound/WebFluxRequestExecutingMessageHandlerTests.java

```diff
@@ -376,4 +384,8 @@ void testMaxInMemorySizeExceeded() {
 				.isEqualTo("Exceeded limit on max bytes to buffer : 1");
 	}
 
+	private void setBeanFactory(WebFluxRequestExecutingMessageHandler handler) {
```

Comment:
Not sure why we need this method, if you simply can do that directly in a line of code.

Reply:
Code has been updated accordingly.